bin/p2-watch/main.go

```diff
@@ -32,32 +41,77 @@ func main() {
 		}
 		*nodeName = hostname
 	}
+	if *podClusters {
+		watchPodClusters(client)
+	} else {
+		podPrefix := kp.INTENT_TREE
+		if *watchReality {
+			podPrefix = kp.REALITY_TREE
+		} else if *hooks {
+			podPrefix = kp.HOOK_TREE
+		}
+		log.Printf("Watching manifests at %s/%s/\n", podPrefix, *nodeName)
 
-	podPrefix := kp.INTENT_TREE
-	if *watchReality {
-		podPrefix = kp.REALITY_TREE
-	} else if *hooks {
-		podPrefix = kp.HOOK_TREE
-	}
-	log.Printf("Watching manifests at %s/%s/\n", podPrefix, *nodeName)
-
-	quit := make(chan struct{})
-	errChan := make(chan error)
-	podCh := make(chan []kp.ManifestResult)
-	go store.WatchPods(podPrefix, types.NodeName(*nodeName), quit, errChan, podCh)
-	for {
-		select {
-		case results := <-podCh:
-			if len(results) == 0 {
-				fmt.Println(fmt.Sprintf("No manifests exist for %s under %s (they may have been deleted)", *nodeName, podPrefix))
-			} else {
-				for _, result := range results {
-					fmt.Println("")
-					result.Manifest.Write(os.Stdout)
+		quit := make(chan struct{})
+		errChan := make(chan error)
+		podCh := make(chan []kp.ManifestResult)
+		go store.WatchPods(podPrefix, types.NodeName(*nodeName), quit, errChan, podCh)
+		for {
+			select {
+			case results := <-podCh:
+				if len(results) == 0 {
+					fmt.Println(fmt.Sprintf("No manifests exist for %s under %s (they may have been deleted)", *nodeName, podPrefix))
+				} else {
+					for _, result := range results {
+						fmt.Println("")
```

Comment:
I realize this is just an indentation change, but I am a bit curious - what was this used for?